content/en/docs/instrumentation/net/_index.md

```diff
@@ -14,21 +14,29 @@ and features.
 OpenTelemetry for .NET supports all officially supported versions of [.NET
 Core](https://dotnet.microsoft.com/download/dotnet-core) and [.NET
 Framework](https://dotnet.microsoft.com/download/dotnet-framework) except for
-.NET Framework 3.5 SP1.
+versions lower than .NET Framework 4.6.1
 
 # Status and Releases
 
 This is the current release status for OpenTelemetry components in this language:
 
 | Tracing | Metrics | Logging |
 | ------- | ------- | ------- |
-| 1.0    | Alpha   | Beta    |
+| Stable  | Beta    | Alpha   |
```

Comment:
unsure about this

Reply:
If you want to see another way of displaying that info, #1116 has an example. In terms of the component status, I think you'd need to double check with the .NET SIG

Reply:
@cijothomas What would be the status of each component?